client/js/config/index.js

```diff
@@ -0,0 +1 @@
+export {default as THEME} from './Theme';
```

Comment:
We do not need this index.js file. Please remove it.

Reply:
Once removed the app crashes because it is being imported into components and screens. This same index file was submitted by Yongmin 3 days ago and it was approved.